lib/committee/drivers.rb

```diff
@@ -17,14 +19,39 @@ def self.driver_from_name(name)
     # @param [String] filepath
     # @return [Committee::Driver]
     def self.load_from_json(filepath)
-      json = JSON.parse(File.read(filepath))
-      load_from_data(json)
+      load_from_data(JSON.parse(File.read(filepath)))
+    end
+
+    # load and build drive from YAML file
+    # @param [String] filepath
+    # @return [Committee::Driver]
+    def self.load_from_yaml(filepath)
+      load_from_data(YAML.load_file(filepath))
+    end
+
+    # load and build drive from file
+    # @param [String] filepath
+    # @return [Committee::Driver]
+    def self.load_from_file(filepath)
+      case File.extname(filepath)
+      when '.json'
+        load_from_json(filepath)
+      when '.yaml', '.yml'
+        load_from_yaml(filepath)
+      else
+        raise "committee filepath option support '.yaml', '.yml', '.json' files only"
```

Comment:
Could you reword this to:

"Committee only supports the following file extensions: '.json', '.yaml', '.yml'"